app/src/main/java/ch/hikemate/app/ui/map/HikeDetailScreen.kt

```diff
@@ -426,7 +443,7 @@ fun DateDetailRow(
                   // saved Date
                   style = MaterialTheme.typography.bodySmall,
                   modifier =
-                      Modifier.clickable { showDatePickerDialog() }
+                      Modifier.clickable { showDatePicker() }
```

Comment:
Request: LGMT, but I noticed that the border of the Box in which this date is is black. Could you change it to onPrimary please or something similar?

Reply:
Thanks